packages/fast-components/src/text-field/text-field.ts

```diff
@@ -0,0 +1,95 @@
+import { attr, FastElement, observable } from "@microsoft/fast-element";
+
+export enum TextFieldAppearance {
+    filled = "filled",
+    outline = "outline",
+}
+
+export enum TextFieldType {
+    email = "email",
+    password = "password",
+    tel = "tel",
+    text = "text",
+    url = "url",
+}
+
+export class TextField extends FastElement {
+    @attr
+    public appearance: TextFieldAppearance = TextFieldAppearance.outline;
```

Comment:
Sort of seems like the enum could be "default" otherwise we have to go back through the code to understand that outline is the default.

Reply:
@JaneChu I'm hesitant to change from this convention as the default could change as the system or design language changes. If the default were to change we would additionally need to add an appearance for `outline`.

Reply:
I suppose what we're asking is if you wanted the default style and it was changed from the outline style to something else, would that be OK. Seems like a question for @bheston

Reply:
Outline and filled are the appearances, it just so happens that outline is what we default to. If a user wanted whatever our default appearance was and our default appearance changed either to filled or something new, they would automatically get that by not setting it. If they do set it to outline or filled, we respect those.

I think the question above is, if someone wants the outline appearance and that is what they get out of the box, they may not set it. If they do not set it and then we change the default to filled that would break their expectation. One potential fix to that would be an additional enum of "default" which then sets to outline; however, if we set that as the default we still have the same problem. If I don't set anything for the appearance it gets set to a default and then that changes. The only consistent way to ensure an explicit style would be to explicitly set it, which I think would be our preference.

Reply:
I think that's good reasoning.